libtransmission/verify.cc

```diff
@@ -94,7 +95,7 @@ void tr_verify_worker::verify_torrent(Mediator& verify_mediator, std::atomic<boo
             if (auto const now = current_time_secs(); last_slept_at != now)
```

Comment:
Ensuring ```sleep_per_seconds_during_verify > 0``` before going into this ```if``` would be better to not affect performance for the as fast as possible verification mode. See also #6112.

Reply:
Good idea! I've addressed this in commit [3f38c73](https://github.com/transmission/transmission/pull/6572/commits/3f38c73c033a700b2fe0c3ac17ba9d66e38dd4ec)